### ISSUE TITLE
Unable to check status / manipulate with camera on 0.1.8.3

### ISSUE THREAD

**Issue by georgo** [BaQs/pyEzviz#40]
So I had issues with login in previous versions, so I upgraded to newset 0.1.8.3. But this version seems still not working in terms of operating with camera.

This ends with fail (please note, ID, email, password and content-length are altered to preserve my privacy):
```bash
$ /usr/local/bin/pyezviz -d -u email@example.com -p redacted camera --serial 123456 status
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): apiieu.ezvizlife.com:443
send: b'POST /v3/users/login/v5 HTTP/1.1\r\nHost: apiieu.ezvizlife.com\r\nUser-Agent: python-requests/2.23.0\r\nAccept-Encoding: gzip, deflate\r\nAccept: */*\r\nConnection: keep-alive\r\nclientType: 3\r\nContent-Length: 123\r\nContent-Type: application/x-www-form-urlencoded\r\n\r\n'
send: b'account=email%example.com&password=redacted&featureCode=c22cb01f8cb83351422d82fad59c8e4e'
reply: 'HTTP/1.1 200 OK\r\n'
header: Content-Encoding: gzip
header: Content-Type: application/json;charset=UTF-8
header: Date: Sun, 04 Apr 2021 08:29:44 GMT
header: Server: Tengine
header: Vary: Accept-Encoding
header: Content-Length: 812
header: Connection: keep-alive
DEBUG:urllib3.connectionpool:https://apiieu.ezvizlife.com:443 "POST /v3/users/login/v5 HTTP/1.1" 200 812
DEBUG:root:Camera loaded
'NoneType' object has no attribute 'get'
Traceback (most recent call last):
  File "/usr/local/bin/pyezviz", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.8/dist-packages/pyezviz/__main__.py", line 251, in main
    client.close_session()
  File "/usr/local/lib/python3.8/dist-packages/pyezviz/client.py", line 866, in close_session
    self._session.close()
AttributeError: 'NoneType' object has no attribute 'close'
```

But it seems login it self is working, calling `devices status` correctly returns all devices in my account!

EU API is my desired destination.

**Comment by georgo** [comment]
FYI: Just tried to downgrade to 0.1.8.2 and it seems bug is not present.

**Comment by RenierM26** [comment]
Hi @georgo,

Found and fixed the problem in 0.1.8.4. Thanks.
